internal/pkg/exec/exec.go

```diff
@@ -0,0 +1,21 @@
+package exec
```

Comment:
Is this file being used? Can't tell because the PR is so big and these names are very generic.

Reply:
Yeah it's used in `internal/cmd/connect/install_utils.go`. I did this so that I can mock `os/exec` in the unit tests.